### PR TITLE
Update source-info.md

### PR DESCRIPTION
**Pull request by JV0812**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/source-info.md [hunk 1]
```diff
@@ -39,4 +45,4 @@ When Sumo Logic detects an issue it is tracked by Health Events. The following t
 
 ## JSON Configuration
 
-Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [how to use JSON to configure Sources](/docs/send-data/use-json-configure-sources), or [Sumo Logic Terraform Resource](https://registry.terraform.io/providers/SumoLogic/sumologic/latest/docs/resources/cloud_to_cloud_source) if you are using Terraform,for more details.
\ No newline at end of file
+Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [how to use JSON to configure Sources](/docs/send-data/use-json-configure-sources), or [Sumo Logic Terraform Resource](https://registry.terraform.io/providers/SumoLogic/sumologic/latest/docs/resources/cloud_to_cloud_source) if you are using Terraform,for more details.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/source-info.md:48]:
```suggestion
Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [How to use JSON to configure sources](/docs/send-data/use-json-configure-sources), or [Sumo Logic Terraform Resource](https://registry.terraform.io/providers/SumoLogic/sumologic/latest/docs/resources/cloud_to_cloud_source) if you are using Terraform,for more details.
```

**Review by kimsauce — COMMENTED** [review]
This page won't load when I previewed `/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/source-info`

**Comment by JV0812** [comment]
Instead of giving an id to the doc, we have used a slug component to this doc. And the same path with slug component is used while referring this doc on all the C2C doc. 
Slug component is used as a trial just to reduce the length of the URL.

**Comment by JV0812** [comment]
> @kimsauce Instead of giving an id to the doc, we have used a slug component to this doc. And the same path with slug component is used while referring this doc on all the C2C doc. Slug component is used as a trial just to reduce the length of the URL.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**